generate/genhcl/genhcl.go

```diff
@@ -34,16 +34,29 @@ import (
 // Is contains parsed and evaluated code on it and information
 // about the origin of the generated code.
 type HCL struct {
-	label     string
-	origin    info.Range
-	body      string
-	condition bool
-	asserts   []config.Assert
+	magicCommentStyle CommentStyle
+	label             string
+	origin            info.Range
+	body              string
+	condition         bool
+	asserts           []config.Assert
 }
 
+// CommentStyle is the configured comment style that must be generated.
+type CommentStyle int
+
+// Comment styles supported.
+const (
+	SlashComment CommentStyle = iota
```

Comment:
In my opinion, if it's an enum, the documentation and examples should say only // and # are allowed.

Reply:
make sense, thanks!